test/e2e/storage/testsuites/driveroperations.go

```diff
@@ -52,22 +52,6 @@ func CreateVolume(driver TestDriver, volType testpatterns.TestVolType) interface
 	return nil
 }
 
-// DeleteVolume deletes volume for test unless dynamicPV test
-func DeleteVolume(driver TestDriver, volType testpatterns.TestVolType, testResource interface{}) {
```

Comment:
hard to figure out so i'm just going to ask. Why do we no longer need this `DeleteVolume` func?

Reply:
Because the new `TestVolume` interface has a `DeleteVolume` method that replaces the standalone `DeleteVolume` function. I had considered leaving an almost empty function like this in place for the sake of consistency:
```
func DeleteVolume(volume TestVolume) {
    if volume != nil {
        volume.DeleteVolume()
   }
}
```

But I decided against it because IMHO it just makes the overall code more complex. This is the same as opening a file: there is `f = open()` followed by ` f.Close` and not some `Close(f)`.